base/plugins.cpp

```diff
@@ -765,13 +774,8 @@ Common::String EngineManager::createTargetForGame(const DetectedGame &game) {
 
 	assert(!domain.empty());
 	if (ConfMan.hasGameDomain(domain)) {
-		int suffixN = 1;
 		Common::String gameid(domain);
```

Comment:
What is the purpose of this variable now?

Reply:
Well, i suppose i could just pass domain as well, yeah.